src/test/resources/soop_sst_trajectory.xml

```diff
@@ -6,68 +6,849 @@
     <virtualDataTable>select trajectory_id as instance_id, * from measurements</virtualDataTable>
     <virtualInstanceTable>select * from indexed_file</virtualInstanceTable>
   </source>
-
   <filename>
-    <sql>select 'file-'||(select id from $instance)||'.nc'
+    <sql>       
+        select
+        'IMOS_SOOP-SST_'
+        ||to_char(min("TIME" AT TIME ZONE 'UTC'), 'YYYYMMDD"T"HH24MISS"Z"')
+        ||'_'
+        ||platform_code
+        ||'_FV01_'
+        ||'END-'
+        ||to_char(max("TIME" AT TIME ZONE 'UTC'), 'YYYYMMDD"T"HH24MISS"Z"')
+        ||'_id-'
+        ||id
+        ||'.nc'
+        from $data
+        group by platform_code, id    
     </sql>
   </filename>
-
-  <globalattributes/>
-
   <dimensions>
     <dimension name="TIME"/>
   </dimensions>
-  <variables>
+  <globalattributes/>
+    <attribute name="file_version" value="'Level 1 - Quality Controlled Data'"/>
```

Comment:
Please check waht are the mandatory attributes in IMOS netcdf user manual : title, abstract, etc are missing.